### PR TITLE
ethiopic

### PR DESCRIPTION
**Pull request by PietroLiuzzo**
ethiopic keyboard layout and language file with combos to support amahric and geez. Translation in language file by Dr. Solomon Gebreyes Beyene. It also contains diacritics for transliteration.

### CODE REVIEW

File: languages/amh.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
+// Keyboard Language
+// please update this section to match this language and email me with corrections!
+// "all" is used here for example purposes, by convention it should be the ISO 639-1 code for the specified language
+// ***********************
```
Comment (Mottie) [inline languages/amh.js]: Would you please give yourself and Dr. Beyene credit in the comment section!

File: languages/amh.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+jQuery.keyboard.language.all = {
```
Comment (Mottie) [inline languages/amh.js]: Please replace the "all" with the language abbreviation for Ethiopic... I tried to find one using [ISO-639-3](https://en.wikipedia.org/wiki/ISO_639-3) language codes, but it isn't listed.

File: languages/amh.js (hunk 1 above)
Comment (PietroLiuzzo) [inline languages/amh.js]: done, I hope is ok!

**Comment by Mottie** [comment]
Please fix the `all` references in the file (2x) with `amh` and I think we'll be good to go!

* in the comments - remove the line starting with `// "all" is used here...`
* `jQuery.keyboard.language.amh = `

**Comment by PietroLiuzzo** [comment]
I failed to find in amh.js the second all.

**Comment by Mottie** [comment]
Hehe sorry, the first was the comment line I asked you to remove.

It looks good, thank you again!

**Comment by PietroLiuzzo** [comment]
thanks to you!

**Pull request merged**